plugins/headlamp-plugin/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@kinvolk/headlamp-plugin",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

It's not a bug fix, we could go with:
```suggestion
  "version": "0.3.0",
```

---

Thanks, I added this into the commit.